dali/pipeline/executor/executor.h

```diff
@@ -438,6 +505,24 @@ void Executor<WorkspacePolicy, QueuePolicy>::RunGPU() {
       TimeRange tr("[Executor] Run GPU op " + op_node.instance_name,
           TimeRange::knvGreen);
       RunHelper(op_node, ws);
+      if (get_memory_stats_) {
+        size_t out_size = 0;
+        size_t reserved_size = 0;
+        auto &stats = gpu_memory_stats_[op_node.instance_name];
+        stats.resize(ws.NumOutput(), {0, 0});
+        for (int i = 0; i < ws.NumOutput(); ++i) {
+          out_size = 0;
+          if (ws.template OutputIsType<CPUBackend>(i)) {
+            out_size = ws.template Output<CPUBackend>(i).nbytes();
+            reserved_size = ws.template Output<CPUBackend>(i).capacity();
+          } else {
+            out_size = ws.template Output<GPUBackend>(i).nbytes();
+            reserved_size = ws.template Output<GPUBackend>(i).capacity();
+          }
+          stats[i].first = std::max(out_size, stats[i].first);
+          stats[i].second = std::max(reserved_size, stats[i].second);
+        }
+      }
```

Comment:
Hi, I'm the same piece of code that you wrote for Mixed Ops. (shame, that it's different than CPU).

Reply:
Done